src/components/Player/network.js

```diff
@@ -14,13 +14,12 @@ import { fetchBackend } from '../../utils/network'
 /*
  * Fetches...
  */
-const fetchStream = async () => {
+const fetchStreamGet = async () => {
```

Comment:
I am also renaming network functions to follow the same naming format of the `API_FOO_BAR` format.